lgsm/functions/info_game.sh

```diff
@@ -356,6 +356,34 @@ fn_info_game_col(){
 	fi
 }
 
+fn_info_game_dayz(){
+	# Config
+	if [ ! -f "${servercfgfullpath}" ]; then
+		servername="${unavailable}"
+		adminpassword="${unavailable}"
+		serverpassword="${unavailable}"
+		maxplayers="${zero}"
+	else
+		servername=$(grep "hostname" "${servercfgfullpath}" | sed -e 's/^[ \t]*//g' -e '/^\//d' -e 's/hostname//g' | tr -d '=\";,:' | sed -e 's/^[ \t]*//' -e 's/[ \t]*$//')
```

Comment:
I would go go more for a single sed instead of this chain of commands.

Example text:
`sed -nr 's/^hostname\s*=\s*"(.*)"\s*;/\1/p' $file`
Example numbers:
`sed -nr 's/^maxPlayers\s*=\s*([0-9]+)\s*;/\1/p'`

Reply:
Thanks for your comments. DayZ engine is very similar to the Arma3 engine. I mostly followed Arma 3 code. This part was copied directly from Arma3. I will try to rewrite it.